src/components/TransactionItemComponent/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { View } from 'react-native';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type Transaction from '@src/types/onyx/Transaction';
+import CategoryCell from './DataCeils/CategoryCell';
+import DateCell from './DataCeils/DateCell';
+import MerchantCell from './DataCeils/MerchantCell';
+import ReceiptCell from './DataCeils/ReceiptCell';
+import TagCell from './DataCeils/TagCell';
+import TotalCell from './DataCeils/TotalCell';
+import TypeCell from './DataCeils/TypeCell';
+
+function TransactionItemComponent({ transactionItem, isLargeScreenWidth, isSelected }:{ transactionItem:Transaction, isLargeScreenWidth:boolean, isSelected:boolean}) {
+    const styles = useThemeStyles();
+    const showTooltip = true;
+    const backgroundColor = isSelected? '#E6E1DA' :'#F8F4F0';
+
+    return (
+        <View style={styles.flex1}>
+                {isLargeScreenWidth ? (
```

Comment:
NAB" tiny nitpick but can we switch this check around? I feel like most of the time in the app we do the pattern of handle small screen first, then handle wide screen